scripts/update-frontend-canister.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
```

Comment:
So that if any of the steps fail, the script will exit with an error

```suggestion

set -e
```

Reply:
nice!